Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -29,6 +29,14 @@ public interface IGitModule
         /// <param name="remoteName">The current remote name.</param>
         /// <param name="newName">The new remote name.</param>
         string RenameRemote(string remoteName, string newName);
+
+        /// <summary>
+        /// Parses the revisionExpression as a git reference and returns an <see cref="ObjectId"/>./>
+        /// </summary>
+        /// <param name="revisionExpression">An expression like HEAD or commit hash that can be parsed as a git reference.</param>
+        /// <returns>An ObjectID representing that git reference</returns>
+        ObjectId RevParse(string revisionExpression);
```

Comment:
Move `[CanBeNull]` from implementation to interface.

https://github.com/gitextensions/gitextensions/blob/de336b7f62c78725c198274de62267590b4b8694/GitCommands/Git/GitModule.cs#L3560-L3574

If you change the name of the param as per @gerhardol's comment, update it in both places.

Reply:
Ok.